wandb/sdk/service/service.py

```diff
@@ -82,7 +85,8 @@ def _launch_server(self) -> None:
             fname = os.path.join(tmpdir, f"port-{pid}.txt")
 
             pid_str = str(os.getpid())
-            exec_cmd_list = [sys.executable, "-m"]
+            executable = self._python_executable or sys.executable
```

Comment:
I think `or "python"` would be reasonable here.  There are other places in the codebase where we assume `sys.executable` will be a python interpreter that we should fix as well so maybe a util / helper is appropriate.

Reply:
ah good catch, will do!